source/_faq/unique_id.markdown

```diff
@@ -4,10 +4,21 @@ description: "This entity does not have a unique ID?"
 ha_category: Configuration
 ---
 
-When you see
+If you try to access the configuration dialog for an entity in your Home Assistant, you might end up seeing this message:
 
-```text
-This entity does not have a unique ID, therefore its settings cannot be managed from the UI
-```
+<p class='img'>
+<img src='/images/faq/faq_no_unique_id.jpg' alt='Screenshot of popup for no unique ID'>
+</p>
```

Comment:
Please use markdown as much as possible.

Reply:
Changed. Had copied that over from other doc pages.